sorting.go

```diff
@@ -71,27 +71,18 @@ func move(db *gorm.DB, value sortingInterface, pos int) (err error) {
 
 	currentPos := value.GetPosition()
 	var results *gorm.DB
-	affectedIDDistinctRecordsCount := 0
-
 	if pos > 0 {
-		query := tx.Table(scope.TableName()).Where("position > ? AND position <= ?", currentPos, currentPos+pos)
-		query.Select("count(distinct(id))").Count(&affectedIDDistinctRecordsCount)
-
-		results = query.UpdateColumn("position", gorm.Expr("position - ?", 1))
+		results = tx.Table(scope.TableName()).Where("position > ? AND position <= ?", currentPos, currentPos+pos).
+			UpdateColumn("position", gorm.Expr("position - ?", 1))
 	} else {
-		query := tx.Table(scope.TableName()).Where("position < ? AND position >= ?", currentPos, currentPos+pos)
-		query.Select("count(distinct(id))").Count(&affectedIDDistinctRecordsCount)
-
-		results = query.UpdateColumn("position", gorm.Expr("position + ?", 1))
+		results = tx.Table(scope.TableName()).Where("position < ? AND position >= ?", currentPos, currentPos+pos).
```

Comment:
when would the pos to be `<= 0` ? I think this condition is not covered by the test?